homeassistant/components/stookwijzer/sensor.py

```diff
@@ -1,66 +1,96 @@
-"""Support for Stookwijzer Sensor."""
+"""Support for Stookwijzer Sensors."""
 
 from __future__ import annotations
 
-from datetime import timedelta
+from collections.abc import Callable
+from dataclasses import dataclass
 
 from stookwijzer import Stookwijzer
 
-from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
-from homeassistant.config_entries import ConfigEntry
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfSpeed
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .const import DOMAIN, StookwijzerState
+from .const import DOMAIN
+from .coordinator import StookwijzerConfigEntry, StookwijzerCoordinator
 
-SCAN_INTERVAL = timedelta(minutes=60)
+
+@dataclass(kw_only=True, frozen=True)
+class StookwijzerSensorDescription(SensorEntityDescription):
+    """Class describing Stookwijzer sensor entities."""
+
+    value_fn: Callable[[Stookwijzer], int | float | str | None]
+
+
+STOOKWIJZER_SENSORS = [
+    StookwijzerSensorDescription(
+        key="windspeed",
+        native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
+        suggested_unit_of_measurement=UnitOfSpeed.BEAUFORT,
+        device_class=SensorDeviceClass.WIND_SPEED,
+        suggested_display_precision=0,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda client: client.windspeed_ms,
+    ),
+    StookwijzerSensorDescription(
+        key="air_quality_index",
+        device_class=SensorDeviceClass.AQI,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda client: client.lki,
+    ),
```

Comment:
These are new and should be in another PR... But, they don't add review load atm (10 lines + snapshots). Happy to take them out though. 🤷